introduction.md

```diff
@@ -6,4 +6,8 @@ Favorite Code School path
 ===========================
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?